src/components/views/elements/ImageView.tsx

```diff
@@ -360,16 +408,17 @@ export default class ImageView extends React.Component<IProps, IState> {
         const showEventMeta = !!this.props.mxEvent;
         const zoomingDisabled = this.state.maxZoom === this.state.minZoom;
 
+        let transition;
+        if (this.animatingLoading) transition = "transform 300ms ease 0s";
```

Comment:
Please add some classes for that transition instead of using `style`, so that we can handle it properly for `@media (prefers-reduced-motion)`. Sorry for nitpicking, but we need to handle a11y! 😄